jobs/alembic/versions/ef5c796d3c00_add_pipeline_engine.py

```diff
@@ -0,0 +1,28 @@
+"""add pipeline_engine
+
+Revision ID: ef5c796d3c00
+Revises: 1408c6c00120
+Create Date: 2024-05-01 14:56:52.306791
+
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ef5c796d3c00"
+down_revision = "1408c6c00120"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "job",
+        sa.Column("pipeline_engine", sa.String(length=255), nullable=True),
```

Comment:
We create a new VARCHAR(255) column, but I did not notice any validation for the field length anywhere in the code. Are we good if user input is longer than 255, I think it will break